url.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "time"
+
+type URL struct {
+	Address     string    `json:"address"`
+	Banned      bool      `json:"banned"`
+	CreatedAt   time.Time `json:"created_at"`
+	ID          string    `json:"id"`
+	Link        string    `json:"link"`
+	Password    bool      `json:"password"`
+	Target      string    `json:"target"`
+	Description string    `json:"description"`
+	UpdatedAt   time.Time `json:"updated_at"`
+	VisitCount  int       `json:"visit_count"`
+}
```

Comment:
we could probably move this to the kutt file and name it `LinkResponse`

Reply:
Righto 👍